parcels/nodes/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .nodelist import *  # noqa
+from .PyNode import *  # noqa
```

Comment:
Adding return to end of file
```suggestion
from .PyNode import *  # noqa

```